samples/net/mqtt/doc/description.rst

```diff
@@ -69,6 +69,18 @@ CONFIG_MQTT_SAMPLE_TRANSPORT_CLIENT_ID - MQTT client ID
 	This configuration sets the MQTT client ID name.
 	If not set, the client ID will default to the modem's IMEI number for nRF91 Series devices, MAC address for nRF70 Series devices, or a random number for Native Posix.
 
+.. _CONFIG_MQTT_SAMPLE_TRANSPORT_USER_NAME:
+
+CONFIG_MQTT_SAMPLE_TRANSPORT_USER_NAME - MQTT client username
+	This configuration sets the MQTT client username for password based authentication. It must be provided if password is provided.
+   Not set by default.
```

Comment:
Alignment here should be a single tab.

Reply:
Yes, I can do this in another PR as all the options would need this fix. I see no problem in rendering though.